caas/specs/types.go

```diff
@@ -52,6 +52,9 @@ type ImageDetails struct {
 // PullPolicy describes a policy for if/when to pull a container image.
 type PullPolicy string
 
+// ContainerConfig describes a config map for a container.
```

Comment:
ContainerConfig describes the config used for setting up a pod container's environment variables.